include/fmt/std.h

```diff
@@ -19,18 +19,43 @@
 #ifdef __cpp_lib_filesystem
 #  include <filesystem>
 
+FMT_BEGIN_NAMESPACE
+
+namespace detail {
+
+template <typename Char>
+void write_escaped_path(basic_memory_buffer<Char>& quoted,
```

Comment:
What is this function for? Avoid string construction if native path's code unit type is the same as Char?

Reply:
Yes.